src/AbstractFixer.php

```diff
@@ -63,6 +63,10 @@ public function __construct()
 
     final public function fix(\SplFileInfo $file, Tokens $tokens)
     {
+        if ($this instanceof ConfigurableFixerInterface && null === $this->configuration) {
```

Comment:
first part true, second not

`configuration === null` means that `->configure(null)` was called only in constructor and then ignored (https://github.com/keradus/PHP-CS-Fixer/blob/8387d21b6979a0a1f30900c5bbb847236bac1fa3/src/AbstractFixer.php#L55), and the `->configure(config)` was not called at all (as if it would, then it would throw exception once more, but this time not silenced)

so `configuration===null` means configuration was not set at all, if it would it either be non-empty or throw an exception